types/domexception/lib/DOMException-impl.d.ts

```diff
@@ -1,11 +1,11 @@
 /** @internal */
 declare class DOMExceptionImpl {
-    constructor(globalObject: object, [message, name]: [string?, string?], privateData?: {});
+  constructor(globalObject: object, [message, name]: [string?, string?], privateData?: {});
 
-    message: string;
-    name: string;
+  message: string;
+  name: string;
 
-    readonly code: string;
+  readonly code: string;
```

Comment:
Undo indentation changes:
```suggestion
    constructor(globalObject: object, [message, name]: [string?, string?], privateData?: {});

    message: string;
    name: string;

    readonly code: string;
```